TestModels/Positional/runtimes/rust/src/operation/get_resource.rs

```diff
@@ -10,13 +10,20 @@ impl GetResource {
     pub fn new() -> Self {
         Self
     }
+
     pub(crate) async fn send(
         client: &crate::client::Client,
         input: crate::operation::get_resource::GetResourceInput,
     ) -> ::std::result::Result<
         crate::operation::get_resource::GetResourceOutput,
         crate::types::error::Error,
     > {
+        if input.name.is_none() {
+    return ::std::result::Result::Err(::aws_smithy_types::error::operation::BuildError::missing_field(
```

Comment:
I feel like going through a aws_smithy_types::error is adding complexity without adding value.
Couldn't we just make a wrap_validation_err out of a String and leave it as that?

Reply:
I think it's valuable: it makes it possible to programmatically handle validation errors, and I can see that being useful for middleware that handles translation of Smithy-modeled types, for example.

As long as displaying these errors gives a good user experience I say we keep them.